website/content/docs/operations/health.mdx

```diff
@@ -55,13 +36,40 @@ The controller health service introduces a single read-only endpoint:
 
 All responses return empty bodies. `GET /health` does not support any input.
 
+## Check the health endpoint using `wget`
+
+The Boundary Enterprise Docker image includes `wget`. It is also available to HCP Boundary users. You can use it to check the health endpoint.
```

Comment:
Does it make sense to rephrase this line a bit ? 
It is also available to HCP Boundary users. 

On-prem customers can check health of both controllers and workers via ops endpoint. HCPb customers can do so for their selfmanaged workers only.